AAXtoMP3

```diff
@@ -959,6 +959,11 @@ do
     fi
   fi
 
+  if [ -f "${cover_file}" ]; then
+    if [ "$loglevel" -gt 1 ]; then log "Adding cover art"; fi
```

Comment:
The `log` function checks the loglevel, no reason to do it here too

Reply:
👍 I actually originally had it that way because I assumed that was how it worked, but then I saw L969 below this: https://github.com/KrumpetPirate/AAXtoMP3/pull/192/files#diff-9baa9ef6dd3590dbfcf3f53a18e7e9b5cc46bd75721fb5452588b820e6f30b98R969
```suggestion
    log "Adding cover art"
```